test/test-ce-currentFragment.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+(() => {
+  /**
+   * Two fragments added via body.appendChild see their document.currentFragment
+   */
+  promise_test(function() { return new Promise(this.step_func((resolve, reject) => {
+    // [setup]
+    var fragment1 = document.createElement('x-fragment');
+    var fragment2 = document.createElement('x-fragment');
+    fragment1.id = 'test-1';
+    fragment2.id = 'test-2';
+
+    fragment1.setAttribute('src', 'ce-currentFragment/test-1.html');
+    fragment2.setAttribute('src', 'ce-currentFragment/test-2.html');
+
+    // [run]
+    document.body.appendChild(fragment1);
+    document.body.appendChild(fragment2);
+
+    Promise.all([
+      fragment1.loaded,
+      fragment2.loaded
+    ]).then(() => {
+      resolve();
```

Comment:
Here you should test if ```document.currentFragment === null```. It's because we want to be sure that after executing a fragment, that variable becomes _null_.